[RAPPS-DB] Add Rohitab API Monitor (api-monitor) to the development category (#250)
reactos/rapps-db: api-monitor.txt

[Section]
Name = API Monitor
Version = v2 (Alpha-r13)
License = EULA
LicenseType = 2

> Description = It is a software that monitors and displays API calls made by applications and services. Its a powerful tool for seeing how applications and services work or for tracking down problems that you have in your own applications.

Reviewer: In the 2nd sentence it should have been `it's` instead of `its`. To fix that a slightly compacted suggestion:
```suggestion
Description = A software to monitor and display API calls made by applications and services. A powerful tool for seeing how applications and services work or for tracking down problems that you have in your own applications.
```

Author: Oups, fixing typos. :+1:

Reviewer: Resolved and approved.
